bundle-size/app.js

```diff
@@ -208,47 +269,79 @@ module.exports = app => {
     };
 
     try {
-      const bundleSizesJson = JSON.parse(
+      const masterBundleSizes = JSON.parse(
         await getBuildArtifactsFile(github, `${baseSha}.json`)
       );
-      const baseBundleSize = bundleSizesJson['dist/v0.js'];
 
-      const bundleSizeDelta = bundleSize - baseBundleSize;
-      const bundleSizeDeltaFormatted = formatBundleSizeDelta(bundleSizeDelta);
+      // Calculate and collect all (non-zero) bundle size deltas and list all
+      // dist files that are missing either from master or from this pull
+      // request.
+      const otherBundleSizeDeltas = [];
+      const missingBundleSizes = [];
+      for (const [file, baseBundleSize] of Object.entries(masterBundleSizes)) {
+        if (file === 'dist/v0.js') {
+          // For now, skip dist/v0.js because it gets special handling.
```

Comment:
Constantize perhaps?

Reply:
Since this essentially amounts to a temporary hack, I prefer to make it _look_ like a hack :)